internal/providers/terraform/azure/testdata/log_analytics_workspace_test/log_analytics_workspace_test.usage.yml

```
@@ -6,3 +6,5 @@ resource_usage:
     monthly_additional_log_data_retention_gb: 30
   azurerm_log_analytics_workspace.log_data_export:
     monthly_log_data_export_gb: 40
+  azurerm_log_analytics_workspace.sentinel_data_ingestion:
+    monthly_sentinel_data_export_gb: 40
```

This usage parameter has a different name. After change the golden file will also change.
```suggestion
    monthly_sentinel_data_ingestion_gb: 40
```